src/support/err.c

```diff
@@ -518,8 +547,13 @@ __wt_panic_func(WT_SESSION_IMPL *session, int error, const char *func, int line,
   WT_GCC_FUNC_ATTRIBUTE((format(printf, 6, 7))) WT_GCC_FUNC_ATTRIBUTE((visibility("default")))
 {
     WT_CONNECTION_IMPL *conn;
+    char concatenated_id_str[WT_MAX_LOG_ID_LENGTH];
     va_list ap;
 
+    /* Use the default WT_DEFAULT_LOG_ID log ID '000' as default identifier for category-specific
```

Comment:
same multi line comment here.

Reply:
Fixed in 0f136343cc588b81524565f6e764cc7d7311d112